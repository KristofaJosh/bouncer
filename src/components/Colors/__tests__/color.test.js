import React from 'react'
import renderer from 'react-test-renderer'

import Color from '../index'

it('renders correctly where there is no background', ()=>{
    const tree = renderer.create(<Color/>).toJSON();
    expect(tree).toMatchSnapshot();
})

it('renders correctly where background is set',()=>{
    const background="primary"
    const tree = renderer.create(<Color background={background}/>).toJSON();
    expect(tree).toMatchSnapshot()
})