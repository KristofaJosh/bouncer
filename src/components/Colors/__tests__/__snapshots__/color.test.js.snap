// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly where background is set 1`] = `
<div
  className="sc-bdVaJa DiWAd"
/>
`;

exports[`renders correctly where there is no background 1`] = `
<div
  className="sc-bdVaJa kEFwkw"
/>
`;
